handbook/strategy/objectives.md

```diff
@@ -0,0 +1,25 @@
+# Launch Pad Objectives
+
+::: warning
+This page is a work in progress.
+:::
+
+We want to create a set of clear, measurable objectives for each school year, in rough order of priority. This helps hold us accountable to our goals, ensure that we are improving the club, and learn from our experiences.
+
+Objectives and results for each year are documented in this page. Each school year is denoted by the year of the fall semester, typically in September, and ends around May of the following year. Objectives have the following format:
+
+- **Objective: some overarching theme we want to work towards**
+  - Result: some measureable result we want to achieve to make progress towards this objective
+
+## 2020
+
+- **Encourage more collaboration between developers and designers**
+  - Every single club member participates in at least 3 discussions on GitHub (issues and pull requests)
```

Comment:
Hmmm Can we possibly change the objective to be more a bit more general towards collaboration? Or maybe it's better to add a point on what we'd like to achieve in collaboration, specifically between developers and designers. (since it's in the objective)

Could be a point about the workshop? Or maybe about how every team will get 1-2 designers

Reply:
Also, love that we have some concrete numbers!

Reply:
> Can we possibly change the objective to be more a bit more general towards collaboration

I want to avoid being too general, since it makes it impossible to measure - GitHub participation is something we want to push for, especially as we go async and remote, and is easy to query/count :) We believe being able to discuss changes and work openly in a written format is also a big part of collaboration - let me know what you think!

> Could be a point about the workshop

I'm not sure a workshop really increases collaboration - you might have a short-term workshop task to work on together, or maybe be able to equip yourself with new skills to use when collaborating, but I think the main thing we want to tackle with this goal is the communication part of collaboration (hence this particular metric)

also cc @renehuang1999 for input if she has any :)

Reply:
Added communication: 1d6e991dc0dc96111f6730bda907570cc46766ce

Another note: traditionally, I feel like designer -> developer communication is okay (because they have to hand off design assets), but often the implementation or WIP stuff doesn't really go back to designers (because there isn't a habit of being aware of a project's progress and discussions on GitHub). Extreme example: https://github.com/ubclaunchpad/ubclaunchpad.com/issues/14 was a sizable design/implementation task that had no input from design (only a verbal checkin), which is far from ideal

cc @sandyklc as well for the design side of things

Reply:
re: collaboration 
That's a good point! I think focusing on GitHub participation is great if our objective is only for "engagement", but it kind of lacks the collaboration part. What if we add something like "Having developers and designers engaged in each stage of the project milestone, communicating with each other to refine and improve the project"? @tarikeshaq @bobheadxi

Reply:
let's discuss an additional measure here in https://github.com/ubclaunchpad/docs/issues/153 !